tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -289,7 +289,7 @@ object TastyFormat {
    *  compatibility, but remains backwards compatible, with all
    *  preceeding `MinorVersion`.
    */
-  final val MinorVersion: Int = 2
+  final val MinorVersion: Int = 4
```

Comment:
During the merge of `language-reference-stable`, git somehow decided there was no conflict here.
I'll make it impossible for ports from the documentation to change anything except the documentation itself and the scaladoc.

Reply:
This should have been minor version `3`.

Reply:
Acctualy, I was wrong, this was only used to release `3.4.0-RC1`. Therefore the version is correct.